src/Core/Models/OrganizationConnectionConfigs/ScimConfig.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Text.Json.Serialization;
+using Bit.Core.Enums;
+
+namespace Bit.Core.Models.OrganizationConnectionConfigs
+{
+    public class ScimConfig
+    {
+        public bool Enabled { get; set; }
```

Comment:
There is a `Enabled` column on the table level for this. Does there also need to be one in the config?

Reply:
According to @MGibson1 , the `Enabled` flag was for a different purpose so I created a lower-level flag for the purpose of SCIM itself, if we need to flatten that back out and push the connection up further we can do that and maybe simplify things a bit.

Reply:
That's fine, I will defer to him.